pkgs/top-level/aliases.nix

```
@@ -643,6 +643,7 @@ mapAliases ({
   v8_3_16_14 = throw "v8_3_16_14 was removed in 2019-11-01: no longer referenced by other packages";
   valadoc = throw "valadoc was deprecated on 2019-10-10: valadoc was merged into vala 0.38";
   vamp = { vampSDK = vamp-plugin-sdk; }; # added 2020-03-26
+  vdirsyncerStable  = throw "vdirsyncerStable is not supported anymore. Use vdirsyncer instead. For more info, see: https://github.com/NixOS/nixpkgs/issues/103026#issuecomment-723428168";
```

```suggestion
  vdirsyncerStable  = vdirsyncer; # added 2020-11-08, see https://github.com/NixOS/nixpkgs/issues/103026#issuecomment-723428168
```

This really shouldn't throw, that makes it hard to have the same configuration for master and stable.

---

Done.